frameworks/PHP/reactphp/reactphp.dockerfile

```diff
@@ -6,25 +6,26 @@ RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /de
 RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php
 RUN apt-get update -yqq > /dev/null && \
     apt-get install -yqq git unzip wget curl build-essential \
-    php8.2-cli php8.2-mbstring php8.2-dev php8.2-xml php8.2-curl php8.2-mysql > /dev/null
+    php8.2-cli php8.2-mbstring php8.2-dev > /dev/null
 
 # An extension is required!
 # We deal with concurrencies over 1k, which stream_select doesn't support.
-RUN wget http://pear.php.net/go-pear.phar --quiet && php go-pear.phar
-#RUN apt-get install -y libuv1-dev > /dev/null
-RUN apt-get install -y libevent-dev > /dev/null
-#RUN pecl install uv-0.2.4 > /dev/null && echo "extension=uv.so" > /etc/php/8.2/cli/conf.d/uv.ini
-RUN pecl install event-3.0.8 > /dev/null && echo "extension=event.so" > /etc/php/8.2/cli/conf.d/event.ini
+RUN apt-get install -yqq libuv1-dev > /dev/null \
+    && pecl install uv-beta > /dev/null \
+    && echo "extension=uv.so" > /etc/php/8.2/cli/conf.d/uv.ini
 
-ADD ./ /reactphp
-WORKDIR /reactphp
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 
 COPY deploy/conf/* /etc/php/8.2/cli/conf.d/
 
-COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
+WORKDIR /reactphp
 
+COPY composer.json .
 RUN composer install --prefer-dist --optimize-autoloader --no-dev --quiet
 
+COPY . .
+
 EXPOSE 8080
 
-CMD php server.php
+ENTRYPOINT ["/usr/bin/php"]
+CMD ["server.php"]
```

Comment:
Shameless plug maybe, but you have tried my ReactPHP focussed docker images? Also comes with JIT turned on out of the box for a nice boost. Here is a multi stage example so you don't have your dev tools in your final image: https://github.com/wyrihaximusnet/docker-php#example-usage

Just a suggestion FYI.

Reply:
I didn't want to change everything too drastically. If you think it would have huge impact on the benchmark, then maybe it'd make sense. However, I think we could/should defer that to another PR.

Reply:
Yeah I get that, hence putting it in as suggestion :)